src/UnknownDeviceErrorHandler.js

```diff
@@ -0,0 +1,32 @@
+import dis from './dispatcher';
+import sdk from './index';
+import Modal from './Modal';
+
+const onAction = function(payload) {
+    if (payload.action === 'unknown_device_error') {
+        var UnknownDeviceDialog = sdk.getComponent("dialogs.UnknownDeviceDialog");
+        Modal.createDialog(UnknownDeviceDialog, {
+            devices: payload.err.devices,
+            room: payload.room,
+            onFinished: (r) => {
+                // XXX: temporary logging to try to diagnose
+                // https://github.com/vector-im/riot-web/issues/3148
+                console.log('UnknownDeviceDialog closed with '+r);
+            },
+        }, "mx_Dialog_unknownDevice");
+    }
+}
+
+let ref = null;
+
+module.exports = {
```

Comment:
Let's not mix ES6 import with commonjs export syntax: `export function startListening() {` would be better (which probably means you'll need to change the places its required to be `import { startListening, stopListening } from '../../UnknownDeviceErrorHandler'`).